src/panels/config/areas/ha-config-area-page.ts

```diff
@@ -58,9 +64,39 @@ class HaConfigAreaPage extends LitElement {
     | AreaRegistryEntry
     | undefined => areas.find((area) => area.area_id === areaId));
 
-  private _devices = memoizeOne(
-    (areaId: string, devices: DeviceRegistryEntry[]): DeviceRegistryEntry[] =>
-      devicesInArea(devices, areaId)
+  private _memerships = memoizeOne(
```

Comment:
```suggestion
  private _memberships = memoizeOne(
```
I assume that is a typo or is there a hidden "meme" here 🤣.